src/Sulu/Bundle/AdminBundle/Command/UpdateBuildCommand.php

```diff
@@ -268,23 +273,46 @@ private function doManualBuild(SymfonyStyle $ui): int
             return static::EXIT_CODE_ABORTED_MANUAL_BUILD;
         }
 
-        $ui->title('Start manual build ...');
+        $conflictingDirectories = [];
+        foreach ([
+             $this->projectDir . '/node_modules',
+             \dirname(__DIR__, 5) . '/node_modules',
```

Comment:
we also use this below to get the `$suluVendorAssetFolder`. maybe define the `$suluVendorAssetFolder` before this?

Reply:
created the variable in the constructor